tst/testall.g

```diff
@@ -0,0 +1,6 @@
+LoadPackage("digraphs");
+if DigraphsTestInstall() and DigraphsTestAll() and DigraphsTestExtreme() then
+  FORCE_QUIT_GAP(0);
```

Comment:
Doesn't `QuitGap(0);` do more or less the same thing in this context?

Reply:
Yes.

Reply:
Actually. `QuitGap` is only introduced in GAP `master` branch, so we can't use it.